data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Sinai Syriac 2A, ff. 1-152v, Gospels",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "1"
+  },
+  "unit_end": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "152"
+  },
+  "work": "https://sinaimanuscripts/library.ucla.edu/work/1",
+  "associated_persons": [
```

Comment:
as with other units, grouping all people together in SDP records and assigning role fields.